docs/diagrams/FindActivityDiagram.puml

```diff
@@ -0,0 +1,16 @@
+@startuml
+start
+:User executes find command;
+
+'Since the beta syntax does not support placing the condition outside the
+'diamond we place it as the true branch instead.
+
+if () then ([valid fields present])
```

Comment:
i think there should be two type of errors, right? like missing fields and invalid fields?

Reply:
The sequence diagram treats all these cases as `else`.

Reply:
But have different error messages, right? no?